src/codegen/sys/cargo_toml.rs

```diff
@@ -82,6 +82,7 @@ fn fill_in(root: &mut Table, env: &Env) {
 
     {
         let build_deps = upsert_table(root, "build-dependencies");
+        // TODO: bump to 1.4 once released
```

Comment:
Shouldn't this become 2.0 because it breaks API?

Reply:
I was planning to do 1.3 -> 1.4 for the API break but you're right, let's go to 2.0 to make it clearer.